sml/metrics/classification/classification_emul.py

```diff
@@ -27,15 +27,86 @@
 from sml.metrics.classification.classification import (
     accuracy_score,
     average_precision_score,
+    balanced_accuracy_score,
     f1_score,
     precision_score,
     recall_score,
     roc_auc_score,
+    top_k_accuracy_score,
 )
 
-
 # TODO: design the enumation framework, just like py.unittest
 # all emulation action should begin with `emul_` (for reflection)
+
+
+def emul_balanced_accuracy(mode: emulation.Mode.MULTIPROCESS):
+    def proc(y_true: jnp.ndarray, y_pred: jnp.ndarray, labels: jnp.ndarray):
+        balanced_score = balanced_accuracy_score(y_true, y_pred, labels)
+        return balanced_score
+
+    def sklearn_proc(y_true, y_pred):
+        balanced_score = metrics.balanced_accuracy_score(y_true, y_pred)
+        return balanced_score
+
+    def check(spu_result, sk_result):
+        np.testing.assert_allclose(spu_result, sk_result, rtol=1, atol=1e-5)
+
+    # Test binary
+    y_true = jnp.array([0, 1, 1, 0, 1, 1])
+    y_pred = jnp.array([0, 0, 1, 0, 1, 1])
+    labels = jnp.array([0, 1])
+    spu_result = emulator.run(proc)(y_true, y_pred, labels)
```

Comment:
you should seal the input tensors before feeding them into the emulator, you can refer to other `emul_` funcs